dask/dataframe/core.py

```diff
@@ -3684,6 +3684,19 @@ def groupby(
         dropna=None,
         **kwargs,
     ):
+        if (
+            self.npartitions > 1
+            and self._partition_type is pd.DataFrame
+            and not PANDAS_GT_150
+            and isinstance(by, list)
+            and len(by) > 1
+            and dropna is False
+        ):
+            warnings.warn(
+                "Grouping by multiple columns with dropna=False is not supported with pandas<1.5.0 and may give "
```

Comment:
should we have a ref to the pandas bug in concat that causes this in the warning.?

Reply:
Wouldn't hurt, but shouldn't hold up this PR, IMO, since the pandas fix is already in.

Reply:
Added a link to https://github.com/dask/dask/issues/8817